### ISSUE TITLE
Elementor duplicator when dragging to container

### ISSUE THREAD

**Issue by thedisturbio** [elementor/elementor#17267]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest development version of Elementor.

## Description

Hi guys testing the container experiment when dragging a widget to the container duplicate is added.

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro) with development versions.
- [x] This bug happens with a vanilla WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by ThrudUK** [comment]
Yes, this happens for me as well.

**Comment by shilo-ey** [comment]
I merged this issue with #17207, which is on the same subject so we could deal with it more efficiently. 
Please feel free to continue the conversation there.

Thanks for your feedback!
